flink-table/flink-table-common/src/main/java/org/apache/flink/table/plan/stats/ColumnStats.java

```diff
@@ -115,20 +189,81 @@ public String toString() {
 		if (max != null) {
 			columnStats.add("max=" + max);
 		}
+		if (maxValue != null) {
```

Comment:
we should only output one `max`, and the old way should have higher priority.

Reply:
`max` and `maxValue` could not be null both, so only one `max` will print if the value is not null